start/client/src/pages/cart.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { RouteComponentProps } from '@reach/router';
+import { BrowserRouterProps} from 'react-router-dom';
```

Comment:
We can actually remove `BrowserRouterProps`, because we don't need them with `react-router-dom` (unless there is a reason I'm unaware of!). So we can remove all of these `BrowserRouterProps`!

```suggestion
```